style.css

```diff
@@ -0,0 +1,129 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	font-size: 62.5%;
+}
+
+@font-face {
+	font-family: walsheim;
+	src: url(GTWalsheimPro-Light.ttf);
+}
+
+body {
+	font-family: walsheim;
+}
+
+nav {
+	position: fixed;
+	width: 100%;
+	top: 0;
```

Comment:
Is there a reason for the top: 0 and left: 0 here? What do you need to achieve by including them?